systers_portal/static/css/style.css

```diff
@@ -85,6 +86,11 @@ body > .container {
   margin: 0 auto;
 }
 
+.add_email {
+  max-width: 350px;
+  padding: 15px;
```

Comment:
Please remove the padding. It adds left padding that makes the `add-email` block unaligned to anything.